Cargo.toml

```diff
@@ -11,18 +11,14 @@ build = "build.rs"
 
 [features]
 default = []
-plugins = ["serde", "serde_derive", "heapsize", "heapsize_derive"]
 
 [build-dependencies]
 cmake = "0.1"
 
 [dependencies]
 euclid = "0.15"
-heapsize = {version = "0.4", optional = true}
-heapsize_derive = {version = "0.1.0", optional = true}
 libc = "0.2"
-serde = {version = "1.0", optional = true}
-serde_derive = {version = "1.0", optional = true}
+serde = {version = "1.0", features=["derive"], optional = true}
 servo-skia = "0.30000008"
```

Comment:
r+ with 0.30000009 here (yet to be published).

Reply:
Well it's not bumping euclid either. I can include it in this PR if you want to

Reply:
Oh right, sorry. Renaming the feature is a breaking change and the skia update is close, so maybe let’s land everything together?

Reply:
Sure